### PR TITLE
feat: virtualenv improvements

### PR DESCRIPTION
**Pull request by scop**
Some virtualenv improvements, see individual commits for details.

### CODE REVIEW

**Comment by elebertus** [comment]
I think more or less the same as https://github.com/justjanne/powerline-go/pull/387/files right?

**Comment by scop** [comment]
That one does not look at `VIRTUAL_ENV_PROMPT` at all, but the part following it seems the same indeed, and more succinctly implemented over there. I'll drop that one from here.

**Comment by scop** [comment]
...except that I won't :) The implementation here falls back to the venv dir even if loading the pyvenv.cfg fails for some reason. For example, does it exist in all venv dirs, e.g. ones created by old versions of `virtualenv`?

**Comment by elebertus** [comment]
When I get a chance I'll compare our changes, we can probably just merge them into one PR since they're both related. :+1:

**Comment by duckworth** [comment]
any chance of getting this merged?

File: segment-virtualenv.go [hunk 1]
```diff
@@ -3,21 +3,28 @@ package main
...
 	"gopkg.in/ini.v1"
 
 	pwl "github.com/justjanne/powerline-go/powerline"
 )
 
 func segmentVirtualEnv(p *powerline) []pwl.Segment {
-	var env string
+	env := os.Getenv("VIRTUAL_ENV_PROMPT")
+	if strings.HasPrefix(env, "(") && strings.HasSuffix(env, ") ") {
+		env = env[1 : len(env)-2]
```
Comment (justjanne) [inline segment-virtualenv.go:16]: I think I'd prefer

```suggestion
		env = strings.TrimPrefix(env, "(")
		env = strings.TrimSuffix(env, ") ")
```

for clarity?

**Review by justjanne — COMMENTED** [review]
Looks great overall, though

File: segment-virtualenv.go (hunk 1 above)
Comment (scop) [reply segment-virtualenv.go]: Sure, done.

**Review by justjanne — APPROVED** [review]

**Pull request merged**